src/nti/contentfragments/schema.py

```diff
@@ -8,6 +8,10 @@
 """
 
 from __future__ import print_function, absolute_import, division
+
+from nti.contentfragments.interfaces import IRstContentFragment
```

Comment:
These should be grouped with the other imports from this package, and use the same syntax.